cups/request.c

```diff
@@ -1151,7 +1151,8 @@ _cupsSetError(ipp_status_t status,	/* I - IPP status code */
  */
 
 void
-_cupsSetHTTPError(http_status_t status)	/* I - HTTP status code */
+_cupsSetHTTPError(http_t	*http,	/* I - HTTP connection to server or @code CUPS_HTTP_DEFAULT@ */
```

Comment:
This code shouldn't support `CUPS_HTTP_DEFAULT` since that's for public APIs to map to the current thread's domain socket.

Reply:
Ok, fixed.